### ISSUE TITLE
[Inputs.json]: Keyboard focus order is incorrect after activating 'Switch to Text' button under the Start Time edit box.

### ISSUE THREAD

**Issue by vagpt** [microsoft/AdaptiveCards-Mobile#86]
### Target Platforms

Android

### SDK Version

2023.12.12.1 (85454887)

### Application Name

Adaptive Cards

### Problem Description

**Test Environment:**
OS: Android 13
Device: S20
Screen Reader: Talkback
AdaptiveCard Android App version: 2023.12.12.1 (85454887)
Tool: Logitech Bluetooth Keyboard

**Repro Steps:**
1. Open Adaptive Cards Visualizer Application.
2. Navigate to sample cards using 'alt+right arrow key' and activate it using 'alt+enter key'.
3. A dropdown would appear. Activate it using 'alt+enter key'.
4. Navigate to Inputs using 'alt+right arrow key' and activate it using 'alt+enter key'.
5. Navigate to the Visual tab item present at the top. A sample card would appear.
6. Navigate to the Start Time edit box inside the sample card using 'alt+right arrow key' and activate it using 'alt+enter key'.
7. Navigate to the 'Switch to Text' button using 'alt+right arrow key' and activate it using 'alt+enter key'
8. Observe the issue.

**Actual Result:**
Keyboard focus order is incorrect after activating 'Switch to Text' button under the Start Time edit box. After acivating the buttun focus order is as follows Switch to Clock button > Cancel Button > OK Button > Hour Edit box > Minute Edit box > Dropdown .

**Expected Result:**
Keyboard focus order should be correct after activating 'Switch to Text' button under the Start Time edit box. It should be as follows Hour Edit box > Minute Edit box > Dropdown > Switch to Clock button > Cancel Button > OK Button > Hour Edit box.

**User Experience:**
Keyboard users would get affected if Keyboard focus order is incorrect after activating 'Switch to Text' button under the Start Time edit box.

**WCAG Reference:**
https://www.w3.org/WAI/WCAG21/Understanding/focus-order

Have feedback to share on Bugs ? please tag bug as “A11yRCA” and add your feedback in the comment section

### Screenshots

https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/c87bc6ca-a4c1-4f11-89a9-e3cf481631d3

![Keyboard focus order is not in logical order](https://github.com/microsoft/AdaptiveCards-Mobile/assets/95913748/50f2dfe8-28bc-4c62-906c-ca677c94b5cd)

### Card JSON

```JSON
NA
```

**Comment by vagpt** [comment]
Please don't assign bug to any Dev until UnderReview Tag is added.
